### PR TITLE
Reorganization of sprint 3

### PR DESCRIPTION
**Pull request by Ivikhostrup**
Review guide:

* Is the intro satisfying? Does it pick up on the second sprint properly?
* Is the review/retro satisfying?
* Does the remainder of the third sprint need some updates in terms of head/tail?
* Is the new section with containerizing the api and database sufficient and correct?

### CODE REVIEW

File: sections/Chapter 5 Sprint 3/retrospective.tex [hunk 1]
```diff
@@ -44,4 +49,5 @@ \subsubsection{Start doing}
     \item Sprints changed to 2-week duration. 4 weeks is too long.\todo{We might not be allowed to change the duration of sprints.}
```
Comment (MrHonket) [inline sections/Chapter 5 Sprint 3/retrospective.tex:49]:
```suggestion
    \item Sprints changed to 2-week duration. 4 weeks is too long.
```

**Review by MrHonket — COMMENTED** [review]
Overall I feel like the reorganization makes sense.
However I feel like the leftover todo's in the retrospective should be fixed before merging.

File: sections/Chapter 5 Sprint 3/docker_implementation.tex [hunk 2]
```diff
@@ -0,0 +1,2 @@
+\subsection{Containerizing the API and database}
+The final major goal of this sprint was to achieve the encapsulation that we described in the second sprint. In order to achieve this we had to add both the API and database to a container each and create a network bridge between the two containers so that they could communicate with each other through a specified port.
```
Comment (MrHonket) [inline sections/Chapter 5 Sprint 3/docker_implementation.tex:2]: Perhaps add a ref to the part where the wanted encapsulation is described?

File: sections/Chapter 5 Sprint 3/retrospective.tex [hunk 3]
```diff
@@ -15,13 +16,17 @@ \subsection{Reflection on the sprint}
...
 
-As the semester courses taught us more about agile methods, we found that we did a few things in a non-conventional way. Planning poker was a great way to get a sense of the scope of the sprint, but we should have used a more structured way to do it. There are a few optimizations to make, such as not using hours to estimate tasks, but rather relative task size.
+As the semester courses taught us more about agile methods, we found that we did a few things in a non-conventional way. Planning poker was a great way to get a sense of the scope of the sprint, but we should have used a more structured way to do it. 
+There are a few optimizations to make, such as not using hours to estimate tasks, but rather relative task size.
 We also learned that doing velocity analysis is a great way to know if team productivity is actually improving, which should be a result of us implementing optimizations and removing impediments.\todo{Actually do velocity analysis.}
 Tasks should also not be marked as complete before they have been shipped to production, as per Scrum convention---a task is not done before it is delivered\todo{Include source - Scrum book by Jeff Sutherland should be fine.}.
```
Comment (MrHonket) [inline sections/Chapter 5 Sprint 3/retrospective.tex:31]: Include the source requested

**Review by MrHonket — CHANGES_REQUESTED** [review]

File: sections/Chapter 5 Sprint 3/CRUD.tex [hunk 4]
```diff
@@ -58,14 +55,14 @@ \subsubsection{WordRatios API}
 If no sources are provided, the database searches for the given words in all sources.
 The Rows of the table matching the terms and sources.
 
-\subsubsection{Schema API}
-The last API we implemented in this iteration is the Schema API which is used to post and retrieve a JSON schema from the database. These are later used to verify JSON objects as input on other APIs. 
-This API concists of two \texttt{GET} methods and a single \texttt{POST} method.
-By creating an API for fetching the validation schemas, we enable easy validation and a common understanding of the structure of input data from the knowledge layer.
+\subsubsection{Schema}
+The last route we implemented in this iteration is the Schema route which is used to post and retrieve a JSON schema from the database. These are later used to verify JSON objects as input on other endpoints. 
+This Schema consists of two \texttt{GET} methods and a single \texttt{POST} method.
```
Comment (Ivikhostrup) [inline sections/Chapter 5 Sprint 3/CRUD.tex:60]:
```suggestion
This schema consists of two \texttt{GET} methods and a single \texttt{POST} method.
```

**Review by mkju19 — APPROVED** [review]

**Review by MrHonket — APPROVED** [review]

**Pull request merged**